src/styles/blocks/root.scss

```diff
@@ -0,0 +1,9 @@
+body {
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` property should be applied globally to all elements using the universal selector `*`, and also include `*:before` and `*:after` to ensure that all elements are consistently sized.